Src/Plugins/Security/SEC101_038.PostgreSqlCredentialsValidator.cs

```diff
@@ -61,6 +67,16 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
             groups.TryGetNonEmptyValue("resource", out FlexMatch resource);
 
             string hostValue = FilteringHelpers.StandardizeLocalhostName(host.Value);
+            string idValue = id.Value;
```

Comment:
>idValue [](http://example.com/codeflow?start=19&length=7)

So you've created a variable here that simply caches id.Value. Why? :) I'm not sure there is a substantive improvement in readability that warrants adding an additional line of code. 

i.e., all this does is transform every 'id.Value' expression to 'idValue'. How does this help? Why did you choose this property to cache and not, say, secret.Value?

Reply:
This was cached because I needed to use the value twice in the code (lines 75 and 89, previously only needed in the latter location), so I cached it to save the round trip on reading the object.